dnf/cli/progress.py

```diff
@@ -46,8 +48,7 @@ def __init__(self, fo=sys.stderr, update_period=0.3, tick_period=1.0, rate_avera
         self.rate_average = rate_average
 
     def message(self, msg):
-        self.fo.write(msg)
-        self.fo.flush()
+        dnf.util._terminal_messenger('write_flush', msg, self.fo)
```

Comment:
Wrong order of comments :), this one should be last. It doesn't make sense otherwise.